src/bsonjs.c

```diff
@@ -85,15 +98,18 @@ PyDoc_STRVAR(dump__doc__,
 "This function wraps `bson_as_json` from libbson.");
```

Comment:
We should update the dump__doc__ and dumps__doc__ docstrings.

Reply:
Done.